rest/admin_api.go

```diff
@@ -50,7 +50,12 @@ func (h *handler) handleCreateDB() error {
 			return err
 		}
 
-		config.cas, err = h.server.bootstrapContext.connection.InsertConfig(*config.Bucket, h.server.config.Bootstrap.ConfigGroupID, config)
+		bucket := dbName
```

Comment:
Shouldn't bucket just be a required field for DB creation?  I didn't think we had any formal mapping between dbName and bucket name.

Reply:
In 2.x if bucket was undefined in a database config, the database name was used.

If you want to make bucket a required field for 3.x that's fine, but this is preserving existing behaivour.

Reply:
Ok, makes sense.